### PR TITLE
Update access link and requests examples

### PR DESCRIPTION
**Pull request by Samk13**
:heart: Thank you for your contribution!

### Description

- Changed expires_at from ISO 8601 DateTime to Date format to fix validation error.
- Removed `created_by.user`, `receiver.community`, and `topic.record` due to Unknown field validation errors.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [ ] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [ ] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/howtos/i18n/).
- [ ] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [ ] I've NOT included third-party code (copy/pasted source code or new dependencies).
    * If you have added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Frontend**

- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines.
- [ ] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines.
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines.

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

File: docs/reference/rest_api_drafts_records.md [hunk 1]
```diff
@@ -1677,7 +1677,7 @@ Content-Type: application/json
 
 {
   "permission": "edit",
-  "expires_at": "2121-03-25T21:06:29.563235"
+  "expires_at": "2024-12-25"
```
Comment (fenekku) [inline docs/reference/rest_api_drafts_records.md:1680]: Ah can you adjust the description in the table above too like:

```
| `expires_at` | string | body | Date string in YYYY-MM-DD format. When the link expires. |
```
?

**Review by fenekku — APPROVED** [review]
Minor addition before merge. Thanks!

File: docs/reference/rest_api_drafts_records.md (hunk 1 above)
Comment (Samk13) [reply docs/reference/rest_api_drafts_records.md:1680]: I would add `expires_at` | string | body | ISO 8601 date format. When the link expires.
and update all others to have either:
- `ISO 8601 date` format
- `ISO 8601 date time` format
and it's clear with the example for each request below each one. WDYT @fenekku ?

Comment (fenekku) [reply docs/reference/rest_api_drafts_records.md]: :100: that'd be great

**Pull request merged**